lib/ansible/modules/network/meraki/meraki_nat.py

```diff
@@ -0,0 +1,679 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Kevin Breit (@kbreit) <kevin.breit@kevinbreit.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: meraki_nat
+short_description: Manage NAT rules in Meraki cloud
+version_added: "2.9"
+description:
+- Allows for creation, management, and visibility of NAT rules (1:1, 1:many, port forwarding) within Meraki.
+
+options:
+    auth_key:
+        description:
+        - Authentication key provided by the dashboard. Required if environmental variable MERAKI_KEY is not set.
+        type: str
+    state:
+        description:
+        - Create or modify an organization.
+        choices: [present, query]
+        default: present
+        type: str
+    net_name:
+        description:
+        - Name of a network.
+        aliases: [name, network]
+        type: str
+    net_id:
+        description:
+        - ID number of a network.
+        type: str
+    org_name:
```

Comment:
`auth_key`, `org_name`, and `org_id` are already documented in the meraki doc_fragment, and should not need to be specified here.

Reply:
I have an inconsistency in the type of `org_id` between the specs and documentation. Meraki announced they're changing the type to a string in the coming weeks anyways, so I will be doing a separate PR across all modules to make `org_id` consistent and correct. The others have been fixed.